src/Api/Issue/StdErrIssueApi.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Api\Issue;
+
+use App\Model\Repository;
+
+/**
+ * This will not write to Github, only output to STDERR. But it will try to read from Github.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class StdErrIssueApi extends GithubIssueApi
```

Comment:
This class is never used. But it was really useful when testing/debugging, so I kept it in the PR.